Sluggy/Sluggy.cs

```diff
@@ -4,29 +4,50 @@
 
 namespace Sluggy
 {
+    /// <summary>
+    ///
+    /// </summary>
     public static class Sluggy
     {
-        public static readonly string DefaultSeparator = "-";
+        private const string DefaultSeparator = "-";
```

Comment:
@sj-freitas can you give your opinion?

Reply:
It should be public, but making it a ```const``` makes sense.